book/A-git-in-other-environments/sections/guis.asc

```diff
@@ -1,112 +1,109 @@
 === Graphical Interfaces
 
 (((GUIs)))(((Graphical tools)))
-Git's native environment is in the terminal.
-New features show up there first, and only at the command line is the full power of Git completely at your disposal.
-But plain text isn't the best choice for all tasks; sometimes a visual representation is what you need, and some users are much more comfortable with a point-and-click interface.
+Ang nitibong environment ng Git ay nasa terminal.
+Ang mga bagong tampok ay ipapakita muna roon, at tanging sa command line ay ang buong lakas ng Git na lubos sa iyong pamamahala.
+Ngunit ang plain text ay hindi ang pinakamahusay na pagpipilian para sa lahat ng mga gawain; kung minsan ang isang biswall na representasyon ay ang iyong kailangan, at ang ilang mga gumagamit ay mas kumportable sa isang interface ng point-and-click.
 
-It's important to note that different interfaces are tailored for different workflows.
-Some clients expose only a carefully curated subset of Git functionality, in order to support a specific way of working that the author considers effective.
-When viewed in this light, none of these tools can be called ``better'' than any of the others, they're simply more fit for their intended purpose.
-Also note that there's nothing these graphical clients can do that the command-line client can't; the command-line is still where you'll have the most power and control when working with your repositories.
+Mahalagang tandaan na ang iba't ibang mga interface ay natatangi para sa iba't-ibang mga workflow.
+Ang ilang mga kliyente ay naglalantad lamang ng maingat na na-curate na subset ng Git functionality, upang suportahan ang isang partikular na paraan ng paggawa na itinuturing ng may-akda na epektibo.
+Kapag tiningnan sa liwanag na ito, wala sa mga tool na ito ay maaaring masabi na ``mas mabuti'' kaysa sa iba pa, ang mga ito ay mas angkop sa kanilang layunin.
+Tandaan din na wala sa mga graphical na kliyente na maaaring gawin na ang command-line client ay hindi kaya; ang command-line ay pa rin kung saan magkakaroon ka ng pinaka makapangyarihan at kontrol kapag gumagawa sa iyong mga repository.
```

Comment:
- you could transliterate `graphical` to `grapikal`
- `command-line na kliyente`